### DISCUSSION TITLE
Proposal: "--help" or "completion ..." should not require a account?

### DISCUSSION THREAD

**Discussion by kastl-ars** [ente-io/ente#3715] [Enhancements]
Hi all,

I discovered Ente recently and started packaging the CLI for openSUSE. During the build, I wanted to check that the version is set properly. Also, I wanted to create the completions for the different shells.

However, in the build environment, no account is configured. Hence both `ente --version` and `ente completion bash` fail:

```
[   25s] + /home/abuild/rpmbuild/BUILDROOT/ente-cli-0.2.1-0.x86_64//usr/bin/ente completion bash
[   25s] 2024/10/15 13:23:02 error getting password from keyring: exit status 1
```

Is there any way to skip the account stuff when executing those commands?

Kind Regards,
Johannes

**Comment by mngshm** [comment]
This error makes sense, I tried ente-cli recently as well and I get prompted for password by the keyring even to run --version. 

I tried adding NOPASSWD permissions to my /etc/sudoers, but that doesn't work as well 😅 tho this was a dirty try
